content/en/docs/reference/glossary/annotation.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Annotation
+---
+
+Annotation refers to a
+[Kubernetes annotation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+attached to a pod. For a list of valid Istio-specific annotations, see
```

Comment:
Annotations can be attached to any resource type, not just pods.

Reply:
fixed